index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Collaboration Task 1 - Survey Form</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script> -->
+      <div id="header">
+        <h1 id="title">Social Media Survey Form</h1>
+        <p id="description">Please fill out accordingly</p>
+      </div>
+
+      <!-- form element -->
+      <form action="" id="survey-form">
+        <div class="form-group">
+          <label for="name" id="name-label">Name</label>
+          <input id="name" class="input-field" type="text" required>
+        </div>
+
+        <div class="form-group">
```

Comment:
To maximize code readability, it would be much better to use semantic HTML tags as much as possible. For example, instead of using a div element to wrap the input sections on the form, a better tag to use would be the section tag. This accurately describes the content of the page and also gives the page a higher SEO score.  😊😊